statuslight_control/config/config.exs

```diff
@@ -3,4 +3,7 @@ use Mix.Config
 config :statuslight_control,
   renderer: IORenderer
 
+config :blinkchain,
+  canvas: {12,1}
+
```

Comment:
The UI was blowing up when trying to load in test mode if this wasn't configured 🤷

Reply:
I probably should have dug in a little more, you _can_ use an `only` keyword on the dep to make it only start in prod, but there's a place in StatuslightControl.Statuslight where the BlinkChain module is being called, so it'd have to be stubbed with another struct in other envs

Reply:
I had some success with the `:runtime` directive from [here](https://hexdocs.pm/mix/master/Mix.Tasks.Deps.html#module-dependency-definition-options) to get the dependency loaded, but not needing its config (or having it try to start outside of prod).  However, I don't think messing with this further is important for this PR.  I will take a look at that later on when I mess with some other project structure things.